website_docs/instrumentation_examples.md

```diff
@@ -13,6 +23,16 @@ Project demonstrating Complete Observability Stack utilizing [Prometheus](https:
 
 Checkout [nodejs-opentelemetry-tempo](https://github.com/mnadeem/nodejs-opentelemetry-tempo) and get started
 
-````bash
+```bash
+docker-compose up --build
+```
+
+### otlp-logzio-fullstack
+
+This is a Fullstack playground of Opentelemetry in Javascript, Javascript in Browsers and Node.js
+
+Checkout [otlp-logzio-fullstack](https://github.com/niko-achilles/otlp-logzio-fullstack) and get started
```

Comment:
i'm not sure we want a tutorial/example like this linked to a vendor in the official doc

Reply:
The tempo one was there before, so I thought I add some more.

maybe ideally this page should only hold official examples from the core and contrib repo and reference the registry for “others” and we ask the authors to move them there

Reply:
I believe tempo is a little different because its an open source project (even though hosted service exist i agree). pinging @dyladan @obecny, WDYT ?

Reply:
Isn't logzio also a hosted service of open source?

Reply:
I'm pretty sure that their product isn't open source, at least from experience and the few search i've just made

Reply:
in my opinion @svrnm the example mentioned  `otlp-logzio-fullstack` can be removed . The purpose is to learn by doing by example in the field of fullstack with server-side-rendering, dynamic-rendering and contribute in my free time to openetelemetry-js.

Reply:
It's linked as a community resource so I think it's fine that it isn't open source as long as it's clear we don't officially endorse it.

Let's talk about this at SIG

Reply:
Talked at SIG. Niko, the original author of the linked repository, told us that this repo is not meant to even be an educational resource. Combined with the concerns mentioned above, please remove the example.

Reply:
I was not able to make it to the SIG yesterday! But let me remove it.

Reply:
Done.